rasa/core/policies/rule_policy.py

```diff
@@ -734,7 +795,7 @@ def _find_action_from_rules(
         else:
             logger.debug("There is no applicable rule.")
 
-        return predicted_action_name
+        return predicted_action_name, best_rule_key or self._create_feature_key(states)
```

Comment:
let's add a comment why `self._create_feature_key(states)` is used a fallback source